packages/Checkbox/stories/examples/CheckboxAlign.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Rule } from "storybook/assets/styles/common.styles";
+import Heading from "@paprika/heading";
+import Checkbox, { checkboxStates } from "../../src/Checkbox";
+
+const { CHECKED, UNCHECKED } = checkboxStates;
+
+const CheckboxExample = props => {
+  const [checkedState, setCheckedState] = React.useState(props.value || UNCHECKED);
+
+  const handleChange = () => setCheckedState(checkedState === CHECKED ? UNCHECKED : CHECKED);
+
+  return (
+    <div>
+      <Heading level={1} displayLevel={2} isLight>
+        Checkbox Alignment Right Example
+      </Heading>
+      <Rule />
```

Comment:
Maybe _this_ should be a component

Reply:
Same as above - https://github.com/acl-services/paprika/pull/199/files#r340298417